setup.py

```diff
@@ -49,11 +49,10 @@
     python_requires=">=3.6",
     package_data={"myst_nb": ["_static/*"]},
     install_requires=[
-        "myst-parser~=0.13.3",
+        "myst-parser>=0.13.5",
```

Comment:
```suggestion
        "myst-parser~=0.13.5",
```

we should still pin to <0.14

Reply:
myst-parser v0.13.4 leads to failing tests:
```
============================================================================================================ short test summary info =============================================================================================================
FAILED tests/test_render_outputs.py::test_metadata_image - TypeError: object of type 'NoneType' has no len()
```

Reply:
yep but it still needs to use `~` not `>`

Reply:
ya I that the `~=` implies "at least as high as this patch release" https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
Oh I see, so your intent here is to exclude any releases `>= 0.14.*`?

Reply:
Yeah we usually try to pin so that minor version stays the same, but patches can be upgraded. At least until we start using major versions and have proper semver 😅

Reply:
Got it - thanks!